src/app/client/package.json

```diff
@@ -61,7 +61,7 @@
     "@project-sunbird/ckeditor-build-classic": "4.1.3",
     "@project-sunbird/client-services": "5.1.2",
     "@project-sunbird/common-consumption": "6.0.0",
-    "@project-sunbird/common-form-elements-full": "6.0.0",
+    "@project-sunbird/common-form-elements-full": "6.0.3",
```

Comment:
hope the plugin you are using has a license to it if not please as the owner to add the same

Reply:
@rajeevsathish Here I am updating the package, it is existing package in sunbirdEd-portal. I kindly request your assistance in following up with the team to ensure the addition of the license to this npm package